app/views/pages/how-to-support-us.html.haml

```diff
@@ -40,11 +53,30 @@
               = image_tag 'how-to-support-us/volunteer.jpg', class: 'rounded-start mw-100'
             .col-md-7
               .card-body
-                %h3.card-title= t('pages.how_to_support_us.volunteer.title')
+                %h3.card-title= t('pages.how_to_support_us.volunteer.volunteer.title')
                 %p.card-text
-                  = t('pages.how_to_support_us.volunteer.description')
-                = mail_to 'hello@codebar.io', t('pages.how_to_support_us.volunteer.link'), subjetc: "I'd like to volunteer with codebar", class: 'btn btn-primary btn-sm text-decoration-none'
+                  = t('pages.how_to_support_us.volunteer.volunteer.description')
+                = mail_to 'hello@codebar.io', t('pages.how_to_support_us.volunteer.volunteer.link'), subjetc: "I'd like to volunteer with codebar", class: 'btn btn-primary btn-sm text-decoration-none'
+
+.py-4.py-lg-5.bg-light#fundraise
+  .container
+    .row.d-flex.align-items-center.mb-5
+      .col-12.col-md-6
+        %h2.h1= t('pages.how_to_support_us.fundraise.title')
+        %p= t('pages.how_to_support_us.fundraise.description_html')
+      .col-12.col-md-6
+        = image_tag 'how-to-support-us/hackney-half.jpeg', alt: t('pages.how_to_support_us.fundraise.image_alt'), class: 'mw-100 rounded'
+    .row
+      .col-12.col-lg-4
+        .ratio{style: '--bs-aspect-ratio: 150%;'}
+          %iframe{title: 'Hackney Half', src: '//runforcharity.com/codebar/hackney-half/register'}
+      .col-12.col-lg-4
+        .ratio{style: '--bs-aspect-ratio: 150%;'}
+          %iframe{title: 'Great South Run', src: '//runforcharity.com/codebar/great-south-run/493220'}
+      .col-12.col-lg-4
+        .ratio{style: '--bs-aspect-ratio: 150%;'}
+          -# Add third sign up form here
```

Comment:
@KimberleyCook we can add a third sign up form here if you have a suggestion for a race.

Reply:
One tiny thing, could we switch around these 2 so they're in date order. Not a blocker, more a preference for me hahaha